classes/class-p4-analytics-values.php

```diff
@@ -265,6 +279,26 @@ function ( $project ) {
 		return [ 'not set' => __( '- Select Global Project -', 'planet4-master-theme-backend' ) ] + array_combine( $names, $names );
 	}
 
+	/**
+	 * Returns local(NRO) project smartsheet options.
+	 *
+	 * @return array
+	 */
+	public function local_projects_options(): array {
+		$local_projects_options = [];
+		if ( $this->local_projects ) {
+			$names                  = array_map(
```

Comment:
This spacing is a bit of a nasty side-effect of the equal sign alignment rule we have, where the result is actually less readable. You can avoid it by leaving a blank line between.